### PR TITLE
Add mapping page for Cloud Foundry provider

### PR DESCRIPTION
**Pull request by fifthposition**
This adds a mapping page (resources and operations) for the Cloud Foundry provider.

### CODE REVIEW

File: reference/providers/cf.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Resource mapping
+
+### Account
+In CF, an account maps to a user account on a CF foundation. This user account is provided via configuration to Spinnaker, and Spinnaker’s privileges are determined by the account’s permissions; for example, Spinnaker can manage apps across all of the orgs and spaces which its account can access.
+
+A Spinnaker instance can use multiple CF accounts to access one or multiple CF foundations.
+
+### Load Balancer
+A Spinnaker load balancer maps to a CF route, which is associated with an app and does not have a separate resource name. The name of the route is the fully-qualified path, including host, domain, port, and path. A load balancer is created as part of a Server Group definition.
```
Comment (dorbin) [inline reference/providers/cf.md]: @fifthposition 
I assume by "app" you're not referring to a Spinnaker application, right?

**Review by dorbin — COMMENTED** [review]
Thanks for the new doc, @fifthposition !

In general, can you link terminology (like route. app, foundation, space) to relevant docs on cloudfoundry.org or elsewhere?

**Comment by fifthposition** [comment]
Thanks @dorbin! Updated based on your feedback.

**Review by dorbin — COMMENTED** [review]
Thanks, @fifthposition! LGTM, pending review by @lwander .

**Comment by fifthposition** [comment]
Hi @dorbin @lwander! Any update here? :-)

File: reference/providers/cf.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Scales a CF app up or down (`cf scale`).
+
+### Enable
+Starts a CF app (`cf start`). Enabling a Server Group in Spinnaker causes a stopped CF app to start.
+
+### Disable
+Stops a CF app (`cf stop`). Disabling a Server Group in Spinnaker causes a CF app to stop.
+
+### Rollback
+Disables the current running Server Group (`cf stop`) and starts the previous version of the Server Group (`cf start`).
```
Comment (lwander) [inline reference/providers/cf.md]: I would reorder these, rollback starts the prior server group before stopping the current one.

File: reference/providers/cf.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+### Server Group
+A Spinnaker Server Group maps to a deployment of a CF app in a specific foundation, org, and space. The CF app is named as `APPNAME_STACK_DETAIL_VERSION`, where `APP_NAME` is the app's name in Spinnaker.  If the app was deployed by Spinnaker, the CF app name will also include an appended `-V` with a version number.
+
+When configuring a Server Group, you can either provide a [manifest](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html) for your CF app or enter the parameters directly into the form (direct).
+
+### Region
+
+A Spinnaker Region maps to a CF [space](https://docs.cloudfoundry.org/concepts/roles.html#spaces). The Region is named as "ORG > SPACE", where "ORG" is the name of the CF [org](https://docs.cloudfoundry.org/concepts/roles.html#orgs) and "SPACE" is the name of the CF space.
+
+### Instance
```
Comment (lwander) [inline reference/providers/cf.md:40]: Please insert an extra newline after the headers for consistency with other pages. Also some markdown rendering fails without the extra line.

File: reference/providers/cf.md (hunk 1 above)
Comment (fifthposition) [reply reference/providers/cf.md]: Fixed. Good catch. Thanks!

File: reference/providers/cf.md (hunk 2 above)
Comment (fifthposition) [reply reference/providers/cf.md]: Done. Thank you!

**Comment by fifthposition** [comment]
@lwander @dorbin Updated (thanks!). OK to go now? :)

**Review by lwander — APPROVED** [review]

**Pull request merged**